acceptance/tests/external_ca_support/jetty_external_root_ca.rb

```diff
@@ -20,18 +20,25 @@
 step "Copy certificates and configuration files to the master..."
 fixture_dir = File.expand_path('../fixtures', __FILE__)
 testdir = master.tmpdir('jetty_external_root_ca')
+backupdir = master.tmpdir('jetty_external_root_ca_backup')
 fixtures = PuppetX::Acceptance::ExternalCertFixtures.new(fixture_dir, testdir)
 
 jetty_confdir = master['puppetserver-confdir']
 
 # Register our cleanup steps early in a teardown so that they will happen even
 # if execution aborts part way.
 teardown do
-  step "Restore /etc/hosts and webserver.conf"
-  on master, "cp -p '#{testdir}/hosts' /etc/hosts"
-  on master, "cp -p '#{testdir}/webserver.conf.orig' '#{jetty_confdir}/webserver.conf'"
+  step "Restore /etc/hosts and puppetserver configs; Restart puppetserver"
+  on master, "cp -p '#{backupdir}/hosts' /etc/hosts"
+  on master, "\\cp -frp #{backupdir}/puppetserver/* #{jetty_confdir}/../"
```

Comment:
@joshcooper this is not a typo. This is to ensure that the actual `cp` command is used rather than the `alias cp='cp -i'` which is present on most systems. If the alias is used the command will get executed in interactive mode which is not what we want here.